src/src/filemanager/hillel/FileNavigator.java

```diff
@@ -49,17 +50,9 @@ public void remove(String path) {
     }
 
     public List<FileData> sortBySize() {
-        List<FileData> fileList = new ArrayList<>();
-        for (List<FileData> pathList : fileMap.values()) {
-
-            fileList.addAll(pathList);
-        }
-        Collections.sort(fileList, new Comparator<FileData>() {
-
-            public int compare(FileData f1, FileData f2) {
-                return Integer.compare(f1.getSize(), f2.getSize());
-            }
-        });
-        return fileList;
+        List<FileData> files = new ArrayList<>();
+        fileMap.values().forEach(files::addAll);
```

Comment:
I didn't mean that. I meant, that you should take `filesMap.values();` and then sort it and then return it. There is no need to add elements to another list.